StructTime.c

```diff
@@ -0,0 +1,42 @@
+#include <stdio.h>
+//struct time is template of Time
+struct time {
+	//struct members
+	int hour,min,sec;
+};
+
+//
+struct time sumTime (struct time t1, struct time t2);
+
+void main(){
+	
+	struct time t1,t2;
+	int sum;
+	
+	printf("Enter current time: \n");
+	scanf("%d%d%d", &t1.hour, &t1.min, &t1.sec);
+	printf("Enter another different time: \n");
```

Comment:
Add a horizontal space here. 
```suggestion

	printf("Enter another different time: \n");
```